src/renderer/scss/_gui.scss

```diff
@@ -92,10 +92,17 @@ input {
     background: var(--input-copyable-bg);
     color: var(--input-copyable-color);
     padding: 10px 16px;
-    border: 1px dashed var(--input-copyable-border);
+    border-bottom: 1px dashed var(--input-copyable-border);
```

Comment:
This shouldn't change. It should be a border on every side

Reply:
Not sure why I did that :upside_down_face: